config/PocketCasts.debug.xcconfig

```diff
@@ -1,5 +1,5 @@
 #include "PocketCasts.base.xcconfig"
 
-PRODUCT_BUNDLE_IDENTIFIER = $PRODUCT_BUNDLE_IDENTIFIER_ROOT
+PRODUCT_BUNDLE_IDENTIFIER = $(PRODUCT_BUNDLE_IDENTIFIER_ROOT)
```

Comment:
Reading this way prevents Fastlane from picking up `PRODUCT_BUNDLE_IDENTIFIER_ROOT` as one of the bundle identifiers to map a provisioning profile to.

Reply:
TIL